# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/child/FileSystemBackgroundRequestHandler.cpp
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/child/FileSystemBackgroundRequestHandler.cpp
--- a/dom/fs/child/FileSystemBackgroundRequestHandler.cpp	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/child/FileSystemBackgroundRequestHandler.cpp	Wed Aug 31 09:34:48 2022 +0000
@@ -7,8 +7,8 @@
 #include "FileSystemBackgroundRequestHandler.h"
 
 #include "fs/FileSystemChildFactory.h"
-#include "mozilla/dom/OriginPrivateFileSystemChild.h"
-#include "mozilla/dom/POriginPrivateFileSystem.h"
+#include "mozilla/dom/FileSystemManagerChild.h"
+#include "mozilla/dom/PFileSystemManager.h"
 #include "mozilla/ipc/BackgroundChild.h"
 #include "mozilla/ipc/Endpoint.h"
 #include "mozilla/ipc/PBackgroundChild.h"
@@ -40,12 +40,12 @@
   }
 
   // Create a new IPC connection
-  Endpoint<POriginPrivateFileSystemParent> parentEndpoint;
-  Endpoint<POriginPrivateFileSystemChild> childEndpoint;
-  MOZ_ALWAYS_SUCCEEDS(POriginPrivateFileSystem::CreateEndpoints(
-      &parentEndpoint, &childEndpoint));
+  Endpoint<PFileSystemManagerParent> parentEndpoint;
+  Endpoint<PFileSystemManagerChild> childEndpoint;
+  MOZ_ALWAYS_SUCCEEDS(
+      PFileSystemManager::CreateEndpoints(&parentEndpoint, &childEndpoint));
 
-  RefPtr<OriginPrivateFileSystemChild> child = mChildFactory->Create();
+  RefPtr<FileSystemManagerChild> child = mChildFactory->Create();
   if (!childEndpoint.Bind(child)) {
     return CreateFileSystemManagerChildPromise::CreateAndReject(
         NS_ERROR_FAILURE, __func__);